app/Policies/LeaderboardEntryPolicy.php

```diff
@@ -14,6 +13,16 @@ class LeaderboardEntryPolicy
 {
     use HandlesAuthorization;
 
+    public function manage(User $user): bool
+    {
+        return $user->hasAnyRole([
+            Role::GAME_HASH_MANAGER,
```

Comment:
Probably don't need game hash manager here. Maybe cheat investigator?

What's the difference between this and the delete logic below? How can an entry be managed other than through deletion?

Reply:
Swapped for cheat investigator.

`manage()` is what Filament uses to determine if a resource is viewable for a user.